source/arm9/hw_math.c

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
You need to add a license header. Copy one from a different file. Also, since you're adding the declaration to `math.h`, rename this to `math.c` for consistency.

Reply:
Added one, let me know if I should use a different one instead. I'm unsure as to the difference between zlibg and bsd licenses.

Reply:
Zlib, so that it's the same as the rest of the library.